File: Osnovne skole/2025/Dzungla/solutions/main.cpp
#include<iostream>
using namespace std;
int main() {
    string s; cin >> s;
    bool good = true;
    for(char c : s) 
        if(c != 'A' && c != 'E' && c != 'I' && c != 'O' && c != 'U') good = false;
    if(good) cout << "DA"; else cout << "NE";
    return 0;
}

File: Osnovne skole/2025/Dzungla/solutions/main.py
sifra = input()

good = True
for c in sifra:
    if c != 'A' and c != 'E' and c != 'I' and c != 'O' and c != 'U':
        good = False

if good:
    print("DA")
else:
    print("NE")